server/bin/www

```diff
@@ -3,93 +3,45 @@
 /**
  * Module dependencies.
  */
-var app = require('../app');
-var debug = require('debug');
-var http = require('http');
-var amqp = require('amqplib/callback_api');
-var io = require('socket.io');
+const app = require('../app');
+const db = require('../libs/database');
+const ws = require('../libs/websocket');
+
+const http = require('http');
+const amqp = require('amqplib/callback_api');
+
+const pg = require('pg');
```

Comment:
The [official doc](https://node-postgres.com/guides/project-structure) recommends to put the DB stuff in a separate file, so it's easier to track down as the project gets bigger.

Reply:
done, moved to startup scripts in libs/*.js